deisctl/config/api.go

```diff
@@ -1,8 +0,0 @@
-package config
```

Comment:
Yeah - I likely confused the config interface with an etcd specific interface since the terms are similar.